Dockerfile.multi-arch

```diff
@@ -0,0 +1,8 @@
+ARG OS="linux"
```

Comment:
why do we need a new Dockerfile vs adding this functionality to the existing one? It seems a bit like a workaround right now that will cause confusion in the future.

Reply:
Agree 100%

Reply:
Done.